tomviz/ModuleThreshold.cxx

```diff
@@ -140,26 +147,79 @@ void ModuleThreshold::addToPanel(QWidget* panel)
     delete panel->layout();
   }
 
-  QHBoxLayout* layout = new QHBoxLayout;
-  panel->setLayout(layout);
-  pqProxiesWidget* proxiesWidget = new pqProxiesWidget(panel);
-  layout->addWidget(proxiesWidget);
+  QVBoxLayout* layout = new QVBoxLayout;
 
-  QStringList fprops;
-  fprops << "SelectInputScalars"
-         << "ThresholdBetween";
+  pqStringVectorPropertyWidget* arraySelection =
+    new pqStringVectorPropertyWidget(
+      this->ThresholdFilter->GetProperty("SelectInputScalars"),
+      this->ThresholdFilter.Get());
+  layout->addWidget(arraySelection);
 
-  proxiesWidget->addProxy(this->ThresholdFilter, "Threshold", fprops, true);
+  pqDoubleRangeSliderPropertyWidget* range =
+    new pqDoubleRangeSliderPropertyWidget(
+      this->ThresholdFilter.Get(),
+      this->ThresholdFilter->GetProperty("ThresholdBetween"));
+  range->setProperty(this->ThresholdFilter->GetProperty("ThresholdBetween"));
```

Comment:
This is the line that fixes the crash.  After discussion with Utkarsh, I'm going to move this line into the constructor of pqDoubleRangeSliderPropertyWidget since it is strange to have to pass the property to both.  So this will be fixed by a ParaView update... and I'll take this out.  We can merge now or hold off for the ParaView change.